conf/conf.go

```diff
@@ -67,11 +73,19 @@ func Parse(path string) (*Conf, error) {
 			trashConf.Excludes = append(trashConf.Excludes, strings.TrimSpace(fields[0][1:]))
 			continue
 		}
+
 		// Otherwise it's an import pattern
 		packageImport := Import{}
 		packageImport.Package = fields[0] // at least 1 field at this point: trimmed the line and skipped empty
+		if len(fields) > 3 {
+			packageImport.Options = parseOptions(fields[3])
+		}
 		if len(fields) > 2 {
-			packageImport.Repo = fields[2]
+			if strings.Contains(fields[2], "=") {
+				packageImport.Options = parseOptions(fields[2])
+			} else {
+				packageImport.Repo = fields[2]
+			}
 		}
 		if len(fields) > 1 {
 			packageImport.Version = fields[1]
```

Comment:
Version is required. We support defining the current project import path using a single field for a line, otherwise we'd probably throw an error if `len(fields) < 2`.